src/Main.java

```diff
@@ -40,6 +43,7 @@ public static void main(String[] args) {
 
             System.out.println(placeHolderUserInput + " kan je uitschrijven en dat ziet het er zo uit: " + translator.translate(placeHolderUserInput));
 
+            //asking if user want to do it again
             System.out.println("\n\nWilt u nog een keer een getal tussen de 0 en de 10 laten uitschrijven?\n1.Ja 2.Nee");
             do {
                 try {
```

Comment:
Interessante aanpak; een try-catch blok. Wist niet dat dit eveneens mogelijk was in Java. Ziet een stuk beter uit dan de huidige if/else die werd aanbevolen.

Reply:
Een try catch is geen vervanging van een if else statement. Je kan het even googlen. Het is een blok waarmee je programma niet gelijk crasht als er een fout optreed in het try blok. Dan gaat hij namelijk gelijk naar het catch blok.